src/streams.h

```diff
@@ -853,10 +853,13 @@ class CBufferedFile
     void FindByte(char ch) {
         while (true) {
             if (nReadPos == nSrcPos)
-                Fill();
-            if (vchBuf[nReadPos % vchBuf.size()] == ch)
-                break;
-            nReadPos++;
+                if (!Fill())
+                    throw std::ios_base::failure("CBufferedFile::FindByte: fail to read");
```

Comment:
Can you kindly use bracket here? It will be much easier to read, and also align the code.